dotnet-desktop-guide/framework/winforms/controls/how-to-determine-page-properties-using-the-pagesetupdialog-component.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Determine Page Properties Using the PageSetupDialog Component"
+description: Learn how to determine page properties like page layout, paper size and other layout choices by using the PageSetupDialog component
```

Comment:
Learn how to determine page properties, like page layout, paper size and other layout choices by using the PageSetupDialog component.